fe/fe-core/src/main/java/org/apache/doris/nereids/rules/implementation/AggregateStrategies.java

```diff
@@ -140,6 +143,48 @@ public List<Rule> buildRules() {
                         return pushdownCountOnIndex(agg, project, filter, olapScan, ctx.cascadesContext);
                     })
             ),
+            RuleType.MINMAX_ON_UNIQUE_TABLE_WITHOUT_PROJECT.build(
```

Comment:
why add additional rule here? why not just change storageLayerAggregate function?

Reply:
this have a filter, and have some others check,
if do this, it will become very complex for storageLayerAggregate